integration/test.go

```diff
@@ -249,5 +276,27 @@ func main() {
 		log.Fatal(err)
 	}
 
+    // check lookup table if truncated
+    info := get_lookup_size(client)
+    if info.Counter != 0 || info.Gauge != 0 || info.Histogram != 0 {
+        log.Fatal("wrong initial lookup_size")
+    }
+    for i := 0; i <= 1000; i++ {
+        resp, err := client.Get(fmt.Sprintf("http://localhost:18003/update?tag=%d", i))
+        if err != nil {
+            log.Fatal(err)
+        }
+        if resp.StatusCode != 200 {
+            log.Fatal("update metric failed")
+        }
+        resp.Body.Close()
+
+        info := get_lookup_size(client)
+        j := i % 1000 + 1;
+        if info.Counter != j || info.Gauge != j || info.Histogram != j {
```

Comment:
I don't think this will actually work, since lookup table is per worker, and clients cannot control which of the workers a request will be processed by.

A simple lua test should be sufficient here I believe.

Reply:
@knyar Sorry, it's my mistake. In fact, I test it in my local env with `worker_processes 1`. :)